src/KDFoundation/core_application.h

```diff
@@ -71,6 +73,7 @@ class KDFOUNDATION_API CoreApplication : public Object
     std::unique_ptr<AbstractPlatformEventLoop> m_platformEventLoop;
     std::unique_ptr<Postman> m_postman;
     std::shared_ptr<spdlog::logger> m_logger;
+    std::shared_ptr<KDBindings::ConnectionEvaluator> m_evaluator;
```

Comment:
Hm, I'm unsure whether this belons into the `CoreApplication` :thinking:
I think having a ConnectionEvaluator as a member of the `EventLoop` could make more sense.

As the idea was pretty much that you could queue events onto any eventloop (I assume usually 1 per thread) this way.

The `connectionEvaluator` access function could still remain in CoreApplication for convenience, and just call `eventLoop().connectionEvaluator()`.

Though I may be misunderstanding how KDFoundations works in that regard.

Reply:
I agree, that would be nicer and make it more easily reusable.